components/primitives/ProgressBar/progress-bar.tsx

```diff
@@ -0,0 +1,27 @@
+import clsx from "clsx";
+import { FC, HTMLProps } from "react";
+
+interface ProgressBarProps {
+  value: string; // Provide the progress value as a number (0-100)
+  props?: HTMLProps<HTMLProgressElement>;
+  className?: string;
+}
+
+const ProgressBar: FC<ProgressBarProps> = ({ value, className, ...props }) => {
+  return (
+    <progress
+      className={clsx(
+        className,
```

Comment:
Another thing comes to mind btw, isn't the aim of this classname prop to enable customizability and if possible default style overrides from parent Component?

If so then why is then is prop's placement first in the class construction function? With how tw-merge works (which is part of cn util), the className string that comes in last to the function wins, while the conflicting classes at the top gets removed.

Hence, the className prop should be at the end of the function yeah, so that it can win over the default classes when necessary and reduce confusion👌✨